agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3708,53 +3708,52 @@ bool slave_thread::slave_fsm(bool &call_slave_select)
             std::get<1>(operationClassesInfo->statically_non_operable_channels_list(0)) =
                 1; // dummy value
 
-            if (!tlvAp->add_operating_classes_info_list(operationClassesInfo)) {
+            if (!ap->add_operating_classes_info_list(operationClassesInfo)) {
                 LOG(ERROR) << "add_operating_classes_info_list failed";
                 return false;
             }
         }
 
         // All attributes which are not explicitely set below are set to
         // default by the TLV factory, see WSC_Attributes.yml
-        auto tlvWscM1 = cmdu_tx.addClass<ieee1905_1::tlvWscM1>();
-        if (tlvWscM1 == nullptr) {
+        auto m1 = cmdu_tx.addClass<ieee1905_1::tlvWscM1>();
```

Comment:
why did you change the name? we usually use the name of the tlv

Reply:
In this case m1 is enough - no need to mention it is a TLV.